templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -35,11 +33,7 @@ import { Transform } from 'stream';
 import { RequestType } from 'google-gax/build/src/apitypes';
 {%- endif %}
 import * as protos from '../../protos/protos';
-{%- if api.legacyProtoLoad %}
```

Comment:
I guess you didn't have chance rebase on the latest master change. You might need to `git pull origin master` and on your feature branch `git merge master`.